From 1debe6b140c37e0aa928559add4abcc95ce54aa2 Mon Sep 17 00:00:00 2001
From: Zdenek Dohnal <zdohnal@redhat.com>
Date: Thu, 26 Sep 2024 23:03:32 +0200
Subject: [PATCH] Default BrowseRemoteProtocols should not include "cups"
 protocol

Works around CVE-2024-47176, the fix will be complete removal of CUPS
Browsing functionality
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index d07b184c..84de9129 100644
--- a/configure.ac
+++ b/configure.ac
@@ -183,9 +183,9 @@ else
 fi
 
 AC_ARG_WITH([browseremoteprotocols],
-	[AS_HELP_STRING([--with-browseremoteprotocols=value], [Set which protocols to listen for in cups-browsed (default: dnssd cups)])],
+	[AS_HELP_STRING([--with-browseremoteprotocols=value], [Set which protocols to listen for in cups-browsed (default: dnssd)])],
 	[with_browseremoteprotocols="$withval"],
-	[with_browseremoteprotocols="dnssd cups"]
+	[with_browseremoteprotocols="dnssd"]
 )
 BROWSEREMOTEPROTOCOLS="$with_browseremoteprotocols"
 AC_SUBST(BROWSEREMOTEPROTOCOLS)
-- 
2.46.2.windows.1

